tools/old/runqlen-pre-6.14.bt

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bpftrace
+/*
+ * runqlen.bt	CPU scheduler run queue length as a histogram.
+ *		For Linux, uses bpftrace, eBPF.
+ *
+ * This is a bpftrace version of the bcc tool of the same name.
+ *
+ * For Linux <= 6.14.
```

Comment:
Shouldn't be?
```suggestion
 * For Linux < 6.14.
```
The change appears in 6.14.

Reply:
Absolutely, thanks for spotting that.